src/applications/mhv-medications/components/RefillPrescriptions/RefillNotification.jsx

```diff
@@ -63,8 +63,8 @@ const RefillNotification = ({ refillStatus, successfulMeds, failedMeds }) => {
         </h2>
         <p>We’re sorry. There’s a problem with our system.</p>
         <p>
-          To request refills, call the pharmacy number on your prescription
-          label.
+          Try requesting your refills again. If it still doesn’t work, contact
+          your VA pharmacy.
```

Comment:
I think it's worthwhile here to update the unit spec to assert that the content is correct, here.